source/making_payments/index.html.md.erb

```diff
@@ -87,37 +122,96 @@ HTTP with test accounts.
+  ...
+}
+```
+
+The response to the ‘Create new payment’ API call will include the URL for the payment. You can get this from either:
 
 * the `Location` response header
 * the `self` section of `_links` in the JSON body
 
-The URL contains the GOV.UK Pay `paymentId`, which is automatically
-generated and identifies the payment. A GET request to this URL
-will return information about the payment and its status.
+The URL contains the GOV.UK Pay `paymentId`, which is automatically generated and identifies the payment. A GET request to this URL will return information about the payment and its status.
 
-Your service will need to store the `paymentId` (or the full URL) for each new
-payment you create. This is so you can check the status of these payments
-later.
+Your service will need to store the `paymentId` (or the full URL) for each new payment you create. This is so you can check the status of these payments later.
 
 Read more about [reporting](/reporting/#report-on-a-payment) and [payment flow](/payment_flow/#how-gov-uk-pay-works).
 
-An example URL:
-`https://publicapi.payments.service.gov.uk/v1/payments/{paymentId}`
-
 The response will also include the ``next_url`` to which you should direct your
 user to complete their payment.
 
 <%= warning_text('Do not expose the `next_url` to anyone other than your paying user. Anyone in possession of the `next_url` could hijack your user’s payment.') %>
 
+### API response fields
+
+#### \_links
+
+The `_links` object includes `href` and `method` fields that you can use to make API requests related to the payment. Use the fields in:
+
+- `self` to [get information about the payment](/reporting/)
+- `events` to [get the payment's events](/reporting/#get-a-payment-s-events)
+- `refunds` to [get all refunds for the payment](/refunding_payments/#get-all-refunds-for-a-single-payment)
+- `cancel` to [cancel the payment](/making_payments/#cancel-a-payment-that-s-in-progress)
+
+The `_links` object also includes a `next_url_post` URL, for example:
+
+```json
+"next_url_post": {
+  "type": "application/x-www-form-urlencoded",
+  "params": "\"description\":\"This is a value for a parameter called description\"",
+  "href": "https://publicapi.payments.service.gov.uk/secure/bb0a272c-8eaf-468d-b3xf-ae5e000d2231",
+  "method": "POST"
+},
+```
+
+To help you test your integration, you can automatically complete the **Enter card details** payment page by making a `POST` API request to the `next_url_post` URL with a `params` object.
+ 
+#### created_date
+
+`created_date` is when you created the payment, in [ISO
+8601 format](https://www.gov.uk/government/publications/open-standards-for-government/date-times-and-time-stamps-standard).
+
+#### payment_provider
+
+`payment_provider` is the name of your payment service provider (PSP), or `sandbox` if you're using a test ('sandbox') account.
+
+#### provider_id
+
+`provider_id` is the unique id that your PSP generated for this payment. 
```

Comment:
It isn't necessarily true that the provider generates this ID. For some providers, e.g. Stripe do, but for Worldpay, we generate an ID and send it to them. This distinction might not be needed for users though, so probably don't need to change this.

Reply:
Ok cool - thanks for the info. Will leave as is for now.

Reply:
I think ID should be capitalised

Reply:
Good spot - will fix.